packages/cardano-services/src/Program/programs/providerServer.ts

```diff
@@ -222,12 +223,33 @@ const serviceMapFactory = (options: ServiceMapFactoryOptions) => {
       });
       return new NetworkInfoHttpService({ logger, networkInfoProvider });
     }, ServiceNames.NetworkInfo),
-    [ServiceNames.TxSubmit]: async () => {
+    [ServiceNames.TxSubmit]: withDbSyncProvider(async (dbPools, cardanoNode) => {
+      if (!genesisData)
+        throw new MissingProgramOption(
+          ServiceNames.NetworkInfo,
+          ProviderServerOptionDescriptions.CardanoNodeConfigPath
+        );
+      const networkInfoProvider = new DbSyncNetworkInfoProvider({
+        cache: {
+          db: getDbCache(),
+          healthCheck: healthCheckCache
+        },
+        cardanoNode,
+        dbPools,
+        epochMonitor: getEpochMonitor(dbPools.main),
+        genesisData,
+        logger
+      });
+      const handleProvider = new KoraLabsHandleProvider({
+        networkInfoProvider,
+        policyId: '', // <--------
```

Comment:
@mkazlauskas  I am not sure where the policyId or serverUrl would come from in this step.

Reply:
Need to add a new arg/variable for provider server in `cli.ts`

Reply:
9a0c19b5dc62fd220c9d1414e8cc59b646a50a97 should fix this